src/picongpu/include/simulation_defines/unitless/speciesAttributes.unitless

```diff
@@ -45,6 +46,22 @@ struct Unit<position<T_Type> >
         return unit;
     }
 };
+template<typename T_Type>
+struct UnitDimension<position<T_Type> >
+{
+    static std::vector<float_64> get()
+    {
+        /* L, M, T, I, theta, N, J
+         *
+         * position is in meter: m
+         *   -> L
+         */
+        std::vector<float_64> unitDimension( 7, 0.0 );
+        unitDimension.at(0) = 1.0; // L^1
```

Comment:
proposal: 
 - to avoid anonym (only id) access of the `unitDimension` vector named values should be used
 - define a enum or more save a struct to access the the slots inside the vector with names
```
struct SiUnits
{
    static BOOST_CONSTEXPR_OR_CONST length = 0;
    static BOOST_CONSTEXPR_OR_CONST mass = 1;
    static BOOST_CONSTEXPR_OR_CONST time = 2;
    static BOOST_CONSTEXPR_OR_CONST electricCurrent = 3;
    static BOOST_CONSTEXPR_OR_CONST thermodynamicTemperatureTheta = 4;
    static BOOST_CONSTEXPR_OR_CONST amountOfSubstance = 5;
    static BOOST_CONSTEXPR_OR_CONST luminousIntensity = 6;
};

// usage:
std::vector<float_64> unitDimension( 7, 0.0 );
unitDimension.at(SiUnits::length) = 1.0; // L^1
```